src/Rates.ts

```diff
@@ -1,92 +1,65 @@
-import BigNumber from "bignumber.js";
+import BN from "bn.js";
 import { Rates as RatesInstance } from "../generated/index";
 import { TransactionObject } from "../generated/types/types";
 import { AbstractContract } from "./AbstractContract";
-import { DECIMALS, DECIMALS_DIV, ONE_ETH_IN_WEI } from "./constants";
-import { InvalidPriceError, ZeroRateError } from "./Errors";
+import { ONE_ETH_IN_WEI } from "./constants";
+import { ZeroRateError } from "./Errors";
 import { EthereumConnection } from "./EthereumConnection";
 import { SET_RATE_GAS_LIMIT } from "./gas";
 import { Transaction } from "./Transaction";
 
 export interface IRateInfo {
-    bnRate: BigNumber /** The rate without decimals */;
-    rate: number /** rate with token decimals */;
+    rate: BN;
     lastUpdated: Date;
 }
 
-export interface IRatesOptions {
-    decimals: Promise<number>;
-    decimalsDiv: Promise<number>;
-    constants: any;
-    ethereumConnection: EthereumConnection;
-}
-
 export class Rates extends AbstractContract {
     // overwrite Contract's  property to have typings
     public instance: RatesInstance; /** web3.js Rates contract instance  */
     private web3: any;
-    private decimals: Promise<number>;
-    private decimalsDiv: Promise<number>;
-    private constants: any;
     private ethereumConnection: EthereumConnection;
 
-    constructor(deployedContractInstance: RatesInstance, options: IRatesOptions) {
+    constructor(deployedContractInstance: RatesInstance, ethereumConnection: EthereumConnection) {
         super(deployedContractInstance);
         this.instance = deployedContractInstance;
-        this.ethereumConnection = options.ethereumConnection;
+        this.ethereumConnection = ethereumConnection;
         this.web3 = this.ethereumConnection.web3;
-        this.decimals = options.decimals;
-        this.decimalsDiv = options.decimalsDiv;
     }
 
-    public async getBnEthFiatRate(currency: string): Promise<BigNumber> {
+    public async getEthFiatRate(currency: string): Promise<BN> {
```

Comment:
This is interface change (just like in `getOrders`, `matchMultipleOrders` etc.). Can we document breaking changes in the PR description as per default practice?
 It would make much easier to list breaking changes in the release notes.

Reply:
I think this is a good practice, but let's do this once we have reached 1.0 :) In the meantime, we'll just sync the changes with clients ourselves.

Reply:
lazy :P 
I added it to the PR description. I think it's useful for ourselves as well.

Reply:
thanks :)